spec/directives/mixin/sass.hrx

```diff
@@ -0,0 +1,5 @@
+<===> content/semicolon/input.sass
+@mixin a
+  @content;
```

Comment:
Why a semicolon?

Reply:
This was porting over a nonconformant test, originally in [spec/non_conformant/sass/semicolon/content.hrx](https://github.com/sass/sass-spec/blob/main/spec/non_conformant/sass/semicolon/content.hrx#L5). Should it be removed?